netkan/netkan/cli/common.py

```diff
@@ -227,6 +228,18 @@ def game(self, game: str) -> Game:
             setattr(self, f'_game_{game_id}', Game(game_id, self))
         return getattr(self, f'_game_{game_id}')
 
+    @property
+    def games(self) -> List[str]:
```

Comment:
Can we call this `game_ids` if it's going to return strings, or have it return `Game`s if it's going to be called `games`?

Reply:
Yep, thanks for keeping on top of the consistency!